sound/soc/sof/topology.c

```diff
@@ -2159,7 +2203,7 @@ static int sof_process_load(struct snd_soc_component *scomp, int index,
  */
 
 static int sof_widget_load_process(struct snd_soc_component *scomp, int index,
-				   struct snd_sof_widget *swidget,
+				   struct snd_sof_widget *swidget, int core,
```

Comment:
could we have all these additions of a new parameter as a separate patch, with the paramter set to zero, and later in another patch we set it a a non-zero value? It'd reduce the size of all this.

Reply:
done